src/components/TodoList/TodoList.jsx

```diff
@@ -1,4 +1,12 @@
+import { TodoInfo } from '../TodoInfo';
+
 // Add the required props
-export const TodoList = () => (
-  <>TodoList markup</>
+export const TodoList = ({ todos }) => (
+  <>
+    <section className="TodoList">
+      {todos.map(todo => (
```

Comment:
The CSS class 'TodoList' is not defined in this file. Ensure that it is imported if it is defined in another file or remove it if it is not being used.